public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -322,6 +322,15 @@ export default class InfluxDatasource extends DataSourceApi<InfluxQuery, InfluxO
       .datasourceRequest(req)
       .then(
         (result: any) => {
+          if (result.data && result.data.results) {
+            const errors = result.data.results.filter((elem: any) => elem.error);
+            if (errors.length > 0) {
+              throw {
+                message: 'Error: ' + errors[0].error,
```

Comment:
Can there be more than 1 error? If so should they be included?
Also we prefix it with `InfluxDB Error` few lines down so I think it would make sense to have the same thing here.

Reply:
Right, changed the prefix. As for the errors, at least in theory there can be many results with different errors - though pretty much everywhere in the app we display only the very first one.